app-modules/student-data-model/src/Filament/Imports/StudentEnrollmentImporter.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2025, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\StudentDataModel\Filament\Imports;
+
+use AdvisingApp\StudentDataModel\Filament\Resources\StudentResource\Pages\Concerns\ImportColumns;
+use AdvisingApp\StudentDataModel\Models\Enrollment;
+use Filament\Actions\Imports\ImportColumn;
+use Filament\Actions\Imports\Importer;
+use Filament\Actions\Imports\Models\Import;
+
+class StudentEnrollmentImporter extends Importer
+{
+    use ImportColumns;
+
+    protected static ?string $model = Enrollment::class;
+
+    public static function getColumns(): array
+    {
+        return [
+            ImportColumn::make('sisid')
+                ->label('Student ID')
+                ->requiredMapping()
+                ->example('########')
+                ->rules([
+                    'required',
+                    'string',
+                    'max:255',
+                ]),
+            ...self::getEnrollmentColumns(),
+        ];
+    }
+
+    public function resolveRecord(): ?Enrollment
+    {
+        $enrollment = new Enrollment();
+        $enrollment->student()->associate($this->data['sisid']);
```

Comment:
I am curious whether or not this is needed. It may not be since the import column is already scoped to the correct column name `sisid`